packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -83,13 +83,23 @@ export function Tooltip({
               <p className={styles.tooltipMessage}>{message}</p>
               <div
                 ref={setArrowRef}
-                style={popperStyles.arrow}
+                style={{
+                  position: "absolute",
+                  left:
+                    floatingStyles.arrow?.x != null
+                      ? `${floatingStyles.arrow?.x}px`
+                      : "",
+                  top:
+                    floatingStyles.arrow?.y != null
+                      ? `${floatingStyles.arrow?.y}px`
+                      : "",
+                }}
```

Comment:
this is a bit different from how Popover manages the arrow

ideally they work the same way so you don't have to understand two systems. a shared hook might even be the way to go. I'll see about making them more similar.

Reply:
thinking on this more, it's both not very intuitive these values are possible nor is it obvious that the `!= null` is **very** important for this to work

maybe a named function to perform the comparison or return the value would help. or even a comment like the Popover version has.

Reply:
I've made it more like the Popover version, and swapped `!=` with `Number.isFinite` they _should_ behave the same but hopefully in a clearer way for a reader?

```
undefined != null
false
null != null
false
0 != null
true
```

and 

```
Number.isFinite(undefined)
false
Number.isFinite(null)
false
Number.isFinite(0)
true
﻿
```

do you think that's any better or nah?

and yeah I'll leave the shared hook, if we want to pursue that, until the cleanup phase.

Reply:
haha fair enough. 

do you have a preference between:

`x !== null && x !== undefined` - verbose but clear
`x != null` - a clever, succinct but sneaky way to check both `undefined` and `null` at the same time, leveraging the fact that `undefined == null` returns `true` 
`Number.isFinite(x)` - well, I guess we just said no thanks to this one

Reply:
good question. looking at the types I don't see how it would be null.... I dunno. I assume they have that for some reason?  they do directly mention the null check in the docs.

I'll just go with the first option and be done with it 😄